data/aerial-photography/hro/index.html

```diff
@@ -151,6 +151,7 @@ <h3 id="UAO2003">
       <h5>Downloads</h5>
       <ul class="dotless">
         <li><a href="https://raster.utah.gov/?cat=HRO%202003%20(30cm)">2003 UAO via Interactive Map </a>
+        <li><a href="https://gis.utah.gov/downloads/uao_metadata">2003 UAO Metadata </a>
```

Comment:
we [don't put gis.utah.gov links]() directly into this website. It should be using a `{% link %}` tag.

```suggestion
        <li><a href="{% link downloads/uao_metadata.htm %}">2003 UAO Metadata </a>
```